src/app/shared/state/entry-wizard.service.ts

```diff
@@ -94,6 +99,8 @@ export class EntryWizardService {
       .subscribe(
         (workflow: BioWorkflow) => {
           this.alertService.detailedSuccess('Workflow ' + repository.gitRegistry + '/' + repository.path + ' has been added');
+          this.workflowService.setWorkflow(<SwaggerBioWorkflow>workflow);
+          this.router.navigateByUrl('/my-workflows' + '/' + workflow.full_workflow_path);
```

Comment:
Nit: I see we have this in a lot of places already, but you could move the slash into the my-workflows string, e.g., `'/my-workflows/' + workflow.full_workflow_path`